packages/pf-codemods/README.md

```diff
@@ -311,22 +311,22 @@ Out:
 <ClipboardCopy  />
 ```
 
-### datalist-remove-ondrags [(#8388)](https://github.com/patternfly/patternfly-react/pull/8388)
+### datalist-remove-props [(#8388)](https://github.com/patternfly/patternfly-react/pull/8388)
 
-We've removed the deprecated `onDragFinish`, `onDragStart`, `onDragMove`, and `onDragCancel` props. This rule will remove them and suggest the user use the DragDrop component.
+We've removed the deprecated `onDragFinish`, `onDragStart`, `onDragMove`, and `onDragCancel`, and `itemOrder` props. This rule will remove them and suggest the user use the DragDrop component.
```

Comment:
Not a blocker, but just to separate the onDrag from itemOrder:

```suggestion
We've removed the deprecated `onDragFinish`, `onDragStart`, `onDragMove`, `onDragCancel`, and `itemOrder` props from DataList.

In addition to removing these props, this rule will suggest using the DragDrop component instead of the `onDrag...` props.
```

Or something along those lines.